src/Estimators/tests/CMakeLists.txt

```diff
@@ -20,6 +20,10 @@ SET(UTEST_NAME unit_test_${SRC_DIR})
 
 SET(SRCS test_accumulator.cpp test_local_energy_est.cpp test_manager.cpp)
 
+IF (NOT REMOVE_TRACEMANAGE)
```

Comment:
What is the concern about this dependency? Also, REMOVE_TRACEMANAGER not TRACEMANAGE,

Reply:
It was to match some conditionals in the code. 
Now i tested compilation with REMOVE_TRACEMANAGER, and the unit test compiles and runs okay.  I'll remove the conditional.